src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -152,6 +152,25 @@ function OptionRowLHN(props) {
     const statusContent = formattedDate ? `${statusText} (${formattedDate})` : statusText;
     const isStatusVisible = Permissions.canUseCustomStatus(props.betas) && !!emojiCode && ReportUtils.isOneOnOneChat(optionItem);
 
+    /**
+     * Checks if the current optionItem is a group chat based on certain conditions.
+     *
+     * @returns {Boolean}
+     */
+    const isGroupChat = function () {
```

Comment:
This does not seem ideal to me, i am going to check if there is a better way

This would mean we have to update this condition every time there might be a new chat report type

Reply:
looking at this:
<img width="314" alt="image" src="https://github.com/Expensify/App/assets/36083550/a2f5747b-5142-4e86-9c80-fd44e76077d6">

I think we could look at chatType === "" && optionItem.type === CONST.REPORT.TYPE.CHAT && !optionItem.isThread and participantAccountIDs.length > 2

Reply:
Didn't realise `chatType` was available here too. Makes sense to use it.

I was initially wary of using `participantAccountIDs.length > 2` because I thought it was possible to create a group chat with just 2 participants but discovered I was wrong after testing. This is perfect, thanks.

Reply:
@akinwale I think we might be changing a lot about Group chats soon so this might change and yes then you can have a group chat with only one member. But that is not the case now

Reply:
I still had to make use of the `!optionItem.isMoneyRequestReport` and `!optionItem.isExpenseRequest` checks in there as some tests fail without them.